runbot_render_logs/models/runbot_repo.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+# © 2015 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import os
+import signal
+
+from openerp.addons.runbot.runbot import run, mkdirs
+
+from openerp.tools import config
+
+from openerp import models
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotRepo(models.Model):
+    _inherit = 'runbot.repo'
+
+    def reload_nginx(self, cr, uid, context=None):
+        settings = {}
```

Comment:
And add # comments where the code change
And # ## comment with section of code is from original method.